tests/DCPS/ManyToMany/DataReaderListener.h

```diff
@@ -64,10 +64,10 @@ class DataReaderListenerImpl
   mutable ACE_Thread_Mutex mutex_;
   const Options& options_;
   DDS::DataReader_var reader_;
-  typedef std::set< ::CORBA::Long> Counts;
-  typedef std::map< ::CORBA::Long, Counts> WriterCounts;
-  typedef std::map< ::CORBA::Long, WriterCounts> ParticipantWriters;
-  typedef std::map<std::string, ParticipantWriters> ProcessParticipants;
+  typedef std::set<::CORBA::Long> Counts;
```

Comment:
```suggestion
  typedef std::set< ::CORBA::Long> Counts;
```
C++03 doesn't allow the `<` and `::` tokens next to each other without whitespace.
Also I usually just use "int" since it's the same thing.  Or drop the initial `::`.